app/models/studio.rb

```diff
@@ -1,3 +1,9 @@
 class Studio < ApplicationRecord
   has_many :movies
+  has_many :movie_actors, through: :movies
+  has_many :actors, through: :movie_actors
+
+  def distinct_actors
+    actors.distinct
+  end
```

Comment:
Woot! Adding another association to make this query easier is definitely the way to go here, great job! It shows how comfortable you are with associations and the `through` relationship between them.

Reply:
Awesome! I was worried this was violating a principle somewhere, but glad to see this is the preferred route.  Much easier to think through for me than doing joins at this point.